### PR TITLE
add drone-gcf plugin

### PR DESCRIPTION
**Pull request by oliver006**
A plugin to manage Google Cloud Functions.

### CODE REVIEW

File: content/drone-plugins/drone-gcf/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
```
Comment (tboerger) [inline content/drone-plugins/drone-gcf/index.md]: It should be placed in content/oliver06/drone-gcf

File: content/drone-plugins/drone-gcf/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+date: 2016-01-01T00:00:00+00:00
+title: Google Cloud Functions
+author: oliver006
+tags: [ publish, google, gcp, gcf ]
+logo: google_gcf.svg
+repo: oliver06/drone-gcf
+image: plugins/gcf
```
Comment (tboerger) [inline content/drone-plugins/drone-gcf/index.md]: It's not published in the plugins hub organization, so this should be oliver006/drone-gcf

File: content/drone-plugins/drone-gcf/index.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
+date: 2016-01-01T00:00:00+00:00
```
Comment (tboerger) [inline content/drone-plugins/drone-gcf/index.md]: Please set a proper date

File: content/drone-plugins/drone-gcf/index.md (hunk 1 above)
Comment (oliver006) [reply content/drone-plugins/drone-gcf/index.md]: done

File: content/drone-plugins/drone-gcf/index.md (hunk 2 above)
Comment (oliver006) [reply content/drone-plugins/drone-gcf/index.md]: done

File: content/drone-plugins/drone-gcf/index.md (hunk 3 above)
Comment (oliver006) [reply content/drone-plugins/drone-gcf/index.md]: done

**Comment by oliver006** [comment]
Thanks for the comments @tboerger

File: content/oliver006/drone-gcf/index.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+date: 2019-02-22T00:00:00+00:00
+title: Google Cloud Functions
+author: oliver006
+tags: [ publish, google, gcp, gcf ]
+logo: google_gcf.svg
+repo: oliver06/drone-gcf
```
Comment (tboerger) [inline content/oliver006/drone-gcf/index.md]: I think there is still a zero missing for proper linking

File: content/oliver006/drone-gcf/index.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+image: oliver06/drone-gcf
```
Comment (tboerger) [inline content/oliver006/drone-gcf/index.md]: I think there is still a zero missing for proper linking

File: content/oliver006/drone-gcf/index.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+---
```
Comment (tboerger) [inline content/oliver006/drone-gcf/index.md]: Missing the version flag

**Comment by oliver006** [comment]
Done - thx @tboerger !

**Comment by tboerger** [comment]
http://plugins.drone.io/oliver006/drone-gcf/ :tada:

**Comment by oliver006** [comment]
Looks good, thanks! 🎉

**Pull request merged**